src/index.html

```diff
@@ -1,12 +1,304 @@
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Фото</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container-card">
+        <div class="container-card_product-card product-card">
+          <img
+            class="image product-card_image"
+            src="./images/imac.jpeg" 
+            alt="image"
+          >
+          <h3 class="product-card_description">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+          <p class="product-card_item-code">Код товара: 195434</p>
+          <div class="product-card_container-mark container-mark" data-qa="card_hover">
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars"></div>
+            <p class="container-mark_reviews">Отзывов: 5</p>
+          </div>
+          <div class="product-card_price price">
+            <div class="price_text-price">Цена:</div>
+            <div class="price_numbers">69 999 грн</div>
+          </div>
+          <div class="product-card_container-buy container-buy">
+            <a
+              href="#"
+              class="container-buy_button-buy"
+              data-qa="card-hover"
+            >
+              КУПИТЬ
+            </a>
+          </div>
+        </div>
+        <div class="product-card">
+          <img
+            class="product-card_image"
+            src="./images/imac.jpeg"
+            alt="image"
+          >
+          <h3 class="product-card_description">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+          <p class="product-card_item-code">Код товара: 195434</p>
+          <div class="product-card_container-mark container-mark">
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars"></div>
+            <p class="container-mark_reviews">Отзывов: 5</p>
+          </div>
+          <div class="product-card_price price">
+            <div class="price_text-price">Цена:</div>
+            <div class="price_numbers">69 999 грн</div>
+          </div>
+          <div class="product-card_container-buy container-buy">
+            <a 
+              href="#"
+              class="container-buy_button-buy"
+              data-qa="card-hover"
+            >
+              КУПИТЬ
+            </a>
```

Comment:
Делай текст uppercase через css
```suggestion
            >
              Купить
            </a>
```